src/tools/kotlin-refinement-generator.ts

```diff
@@ -60,17 +60,28 @@ class KotlinRefinementGenerator extends RefinementExpressionVisitor<string> {
   }
   visitBuiltIn(expr: BuiltIn): string {
     // TODO: Double check that millis are the correct default units.
-    if (expr.value === 'now()') {
-      return `System.currentTimeMillis()`;
+    switch (expr.value) {
+      case 'now': return `System.currentTimeMillis().asExpr()`;
```

Comment:
I don't think this will do what you want, since this will bake in a literal number into the DSL when the object (e.g. Schema) is constructed, so it will be the 'now' of when Schema() was called.  I am working on a much bigger PR that adds a FunctionExpression node to the DSL that will allow you to create a first class now() function. Perhaps I will separate it out and land it for you first.

Reply:
Oh you're totally right, thanks Ray. Can you tap me on the shoulder when that work is done?

Do you think we should land this, knowing about the bug (and with a plan for how to fix it) or should we hold off?

Reply:
I think we should go ahead and land this, maybe with a warning note / TODO somewhere about the behaviour of `now`.

Reply:
You can land it, my PR is about to land, and then I'll go back and fix this by adding the now() function for you.

Reply:
Also calling System.currentTimeMillis() makes the code non-core non-portable too :)

Reply:
FYI @Cypher1 https://github.com/PolymerLabs/arcs/pull/5868